src/components/login/login.jsx

```diff
@@ -1,18 +1,63 @@
-import React from "react";
+import React, { useState } from "react";
 import './index.css';
-import { Link, useNavigate} from 'react-router-dom';
+import { Link, useNavigate,ReactDOM} from 'react-router-dom';
 import logo from '../../assets/Images/logo.png';
 
 function LoginModule(){
 
+    /// Error Message
+    const [errorMessages, setErrorMessages] = useState({});
+
+    /// Success log in
+    const [isSubmitted, setIsSubmitted] = useState(false);
+
+    // User info
+    const database = [
+        {
+          username: "bryan",
+          password: "123"
+        }
+    ];
+
+    // Error message
+    const errors = {
+        uname: "Username or Password invalid",
+        pass: "Username or Password invalid"
+    };
+
     let navigate = useNavigate();
 
-    const onSubmitHandler = (e) => {
-        e.preventDefault();
+    const handleSubmit = (event) => {
+        //Prevent page reload
+        event.preventDefault();
         navigate("/");
```

Comment:
This should be done only after the username and password match the data, if put here the error did not appear on my side